.gitignore

```diff
@@ -15,4 +15,5 @@
 .cxx
 local.properties
 /gcp/scripts/key.json
-__pycache__/
\ No newline at end of file
+__pycache__/
+google-services.json
```

Comment:
you need to go to https://console.firebase.google.com/u/1/project/stoked-depth-428423-j7/overview and click on the android project at the top, and then go down to download the google-services.json file and save to the app/ directory in this repo for the emulator to work.
![image](https://github.com/user-attachments/assets/9330f741-dcb4-4775-a547-82e648e4e1d9)
![image](https://github.com/user-attachments/assets/2d0d1601-e41d-474c-ae41-8e1c10743204)